service/src/test/java/tds/session/repositories/impl/SessionRepositoryImplIntegrationTests.java

```diff
@@ -3,20 +3,19 @@
 import org.junit.Test;
 import org.junit.runner.RunWith;
 import org.springframework.beans.factory.annotation.Autowired;
-import org.springframework.boot.test.SpringApplicationConfiguration;
+import org.springframework.boot.test.context.SpringBootTest;
 import org.springframework.test.context.junit4.SpringJUnit4ClassRunner;
 
 import java.util.Optional;
 import java.util.UUID;
 
 import tds.session.Session;
-import tds.session.SessionServiceApplication;
 import tds.session.repositories.SessionRepository;
 
 import static org.assertj.core.api.Assertions.assertThat;
 
 @RunWith(SpringJUnit4ClassRunner.class)
-@SpringApplicationConfiguration(classes = SessionServiceApplication.class)
+@SpringBootTest
```

Comment:
This is in the repository integration test.  The issue I am having with @SpringBootTest is in the controller integration tests when RestAssured needs to connect.  I can get it running in IntelliJ with @SpringBootTest(webEnvironment = DEFINED_PORT), but it fails when running the integration tests in Maven.